code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -195,10 +195,9 @@
 	icon_state = "battlemage"
 	item_state = "battlemage"
 	recharge_rate = 0
+	max_charges = INFINITY
 	current_charges = 15
-	recharge_cooldown = INFINITY
```

Comment:
Don't we not want the battlemage armour recharging over time?

Reply:
Already handled by the zero `recharge_rate` 2-3 lines above.